test/files/pos/CharSequence-implicits.scala

```diff
@@ -0,0 +1,4 @@
+object Test extends App {
+  Array('a', 'b'): CharSequence
+  IndexedSeq('a', 'b'): CharSequence
+}
```

Comment:
OK, I take it back, my comment is funny again. Thanks for the chuckle! I also appreciate the air of mystery. It's kind of like Penn and Teller, humor and magic.